recipe/meta.yaml

```diff
@@ -38,6 +38,7 @@ outputs:
         - python-constraint
         - cachetools
         - cached-property
+        - typing_extensions
```

Comment:
Fully support adding this in since explicit dependencies are worthwhile, but for the record it was already there as an implicit dependency. In one of my development environments:


```console
$ mamba repoquery whoneeds -t typing-extensions          14:35:08

                  __    __    __    __
                 /  \  /  \  /  \  /  \
                /    \/    \/    \/    \
███████████████/  /██/  /██/  /██/  /████████████████████████
              /  / \   / \   / \   / \  \____
             /  /   \_/   \_/   \_/   \    o \__,
            / _/                       \_____/  `
            |/
        ███╗   ███╗ █████╗ ███╗   ███╗██████╗  █████╗
        ████╗ ████║██╔══██╗████╗ ████║██╔══██╗██╔══██╗
        ██╔████╔██║███████║██╔████╔██║██████╔╝███████║
        ██║╚██╔╝██║██╔══██║██║╚██╔╝██║██╔══██╗██╔══██║
        ██║ ╚═╝ ██║██║  ██║██║ ╚═╝ ██║██████╔╝██║  ██║
        ╚═╝     ╚═╝╚═╝  ╚═╝╚═╝     ╚═╝╚═════╝ ╚═╝  ╚═╝

        mamba (1.1.0) supported by @QuantStack

        GitHub:  https://github.com/mamba-org/mamba
        Twitter: https://twitter.com/QuantStack

█████████████████████████████████████████████████████████████


Executing the query typing-extensions



typing-extensions[4.4.0]
  ├─ aiosqlite[0.18.0]
  │  └─ ypy-websocket[0.8.2]
  │     └─ jupyter_server_ydoc[0.6.1]
  │        └─ jupyterlab[3.6.0]
  ├─ platformdirs[2.6.2]
  │  ├─ black[22.12.0]
  │  ├─ jupyter_core[5.2.0]
  │  │  ├─ jupyterlab already visited
  │  │  ├─ ipykernel[6.21.0]
  │  │  │  ├─ jupyter[1.0.0]
  │  │  │  │  └─ yank[0.25.2]
  │  │  │  │     └─ openff-evaluator[0.3.11]
  │  │  │  ├─ jupyter_console[6.5.1]
  │  │  │  │  └─ jupyter already visited
  │  │  │  ├─ nbclassic[0.4.8]
  │  │  │  │  ├─ jupyterlab already visited
  │  │  │  │  └─ notebook[6.5.2]
  │  │  │  │     ├─ jupyterlab already visited
  │  │  │  │     ├─ jupyter already visited
  │  │  │  │     ├─ openff-toolkit[0.12.0]
  │  │  │  │     │  └─ openff-interchange[0.2.3]
  │  │  │  │     └─ widgetsnbextension[3.6.1]
  │  │  │  │        └─ ipywidgets[7.7.3]
  │  │  │  │           ├─ jupyter already visited
  │  │  │  │           ├─ openff-toolkit already visited
  │  │  │  │           └─ nglview[3.0.3]
  │  │  │  │              └─ qcportal[0.15.8]
  │  │  │  │                 └─ openff-qcsubmit[0.4.1]
  │  │  │  ├─ notebook already visited
  │  │  │  ├─ nbval[0.9.6]
  │  │  │  └─ qtconsole-base[5.4.0]
  │  │  │     └─ qtconsole[5.4.0]
  │  │  │        └─ jupyter already visited
  │  │  ├─ nbclassic already visited
  │  │  ├─ notebook already visited
  │  │  ├─ qtconsole-base already visited
  │  │  ├─ jupyter_client[8.0.2]
  │  │  │  ├─ ipykernel already visited
  │  │  │  ├─ jupyter_console already visited
  │  │  │  ├─ nbclassic already visited
  │  │  │  ├─ notebook already visited
  │  │  │  ├─ nbval already visited
  │  │  │  ├─ qtconsole-base already visited
  │  │  │  ├─ jupyter_server[2.2.0]
  │  │  │  │  ├─ jupyterlab already visited
  │  │  │  │  ├─ nbclassic already visited
  │  │  │  │  ├─ jupyter_server_fileid[0.6.0]
  │  │  │  │  │  └─ jupyter_server_ydoc already visited
  │  │  │  │  ├─ jupyterlab_server[2.19.0]
  │  │  │  │  │  └─ jupyterlab already visited
  │  │  │  │  └─ notebook-shim[0.2.2]
  │  │  │  │     └─ nbclassic already visited
  │  │  │  └─ nbclient[0.7.2]
  │  │  │     └─ nbconvert-core[7.2.9]
  │  │  │        ├─ notebook already visited
  │  │  │        ├─ jupyter_server already visited
  │  │  │        ├─ nbconvert[7.2.9]
  │  │  │        │  ├─ jupyter already visited
  │  │  │        │  └─ nbclassic already visited
  │  │  │        └─ nbconvert-pandoc[7.2.9]
  │  │  │           └─ nbconvert already visited
  │  │  ├─ jupyter_server already visited
  │  │  ├─ nbclient already visited
  │  │  ├─ nbconvert-core already visited
  │  │  └─ nbformat[5.7.3]
  │  │     ├─ nbclassic already visited
  │  │     ├─ notebook already visited
  │  │     ├─ nbval already visited
  │  │     ├─ jupyter_server already visited
  │  │     ├─ nbclient already visited
  │  │     └─ nbconvert-core already visited
  │  └─ virtualenv[20.19.0]
  │     └─ pre-commit[3.0.4]
  ├─ pydantic[1.10.5]
  │  ├─ qcportal already visited
  │  ├─ openff-qcsubmit already visited
  │  ├─ forcefield-utilities[0.2.1]
  │  │  └─ gmso[0.9.1]
  │  │     └─ foyer[0.11.3]
  │  ├─ gmso already visited
  │  ├─ openff-evaluator-base[0.3.11]
  │  │  └─ openff-evaluator already visited
  │  ├─ openff-models[0.0.4]
  │  └─ qcelemental[0.25.1]
  │     └─ qcportal already visited
  └─ sqlalchemy[2.0.4]
     ├─ openff-recharge[0.4.0]
     └─ rdkit[2022.09.1]
        ├─ openff-toolkit already visited
        ├─ openff-qcsubmit already visited
        ├─ openff-evaluator-base already visited
        ├─ openff-recharge already visited
        ├─ cmiles[0.1.6]
        └─ mbuild[0.16.0]